rq/worker.py

```diff
@@ -223,7 +223,11 @@ def __init__(
         serializer=None,
     ):  # noqa
 
-        connection = self._set_connection(connection, default_worker_ttl)
+        self.default_result_ttl = default_result_ttl
+        self.worker_ttl = default_worker_ttl
```

Comment:
Sorry, one more nitpicky thing, `worker_ttl` makes it sound like the worker itself only lives for a certain amount of time (which it doesn't). So if we set 60 seconds as TTL, this makes it seems like the worker would only live for 60 seconds.

Can we rename this to `self.dequeue_ttl = default_worker_ttl`?

I have put renaming `Worker(default_worker_ttl)` in the list of things to change in version 2.0, since this won't be backwards compatible https://github.com/rq/rq/issues/1800

Reply:
`def heartbeat` uses this ttl as well, as well as `register_birth`. Are you sure about this?

Reply:
Yes, this should be a simple find and replace situation so should be ok.

Reply:
I am not talking about the rename technically 😅 , that's not a problem - I did that already from `default_worker_ttl` to `worker_ttl`.
I am meaning semantically, calling it `dequeuet_ttl`, when `heartbeat` and `register_birth` using it is a bit confusing isn't it? Is it right for heartbeat and birth to use "dequeue_ttl" as their reference?

Reply:
Ah I see, you have a point. I'll merge this in.